docs/socket-groups.md

```diff
@@ -153,37 +175,47 @@ protection, the mechanism should quickly detect a link as broken so
+
 As there could be various ways as to how to implement balancing
 algorithm, there's a framework provided to implement various methods,
 and two algorithms are currently provided:
 
-1. `plain` (default). This is a simple round-robin - next link selected
-to send the next packet is the oldest used so far.
-
-2. `window`. This algorithm is performing cyclic measurement of the
+1. `fixed`. In this algorithm you specify the share that is given to
+particular link manually through the `weight` parameter. The weight
+value in case of this algorithm in balancing groups defines how much of
+a share is given to this link. Important here is that you can easily
+think of the weight values as a percentage of load burden for particular
+link - however in reality the share of the load is calculated as a
+percentage that particular link's weight comprises among the sum of all
+weight values. Additionally, a value of 0 is special and is translated
+into a weight that would make it equal to an average of all weights.
+Be careful here with the non-established and broken links. For example,
+if you have 3 links with weight 10, 20 and 30, it results in a load
+balance of 16.6%, 33.3% and 50% respectively. However if the second link
+gets broken, there are then 2 links with 10 and 30, which results in
+load balance of 25% and 75% respectively. Keep in mind that it's up
+to the application to keep the minimum links allowed and break the
+group link that is unable to withstand the load.
+
+2. `window` (default). This algorithm is performing cyclic measurement of the
 minimum flight window and this way determines the "cost of sending"
 of a packet over particular link. The link is then "paid" for sending
 a packet appropriate "price", which is collected in the link's "pocket".
 To send the next packet the link with lowest state of the "pocket" is
 selected. The "cost of sending" measurement is being repeated once per
 a time with a distance of 16 packets on each link.
 
-There are possible also other methods and algorithms, like:
-
-a) Explicit share definition. You declare, how much bandwidth you expect
-the links to withstand as a percentage of the signal's bitrate. This
-shall not exceed 100%. This is merely like the above Window algorithm,
-but the "cost of sending" is defined by this percentage.
-
-b) Bandwidth measurement. This relies on the fact that the current
-sending on particular link should use only some percentage of its
-overall possible bandwidth. This requires a reliable way of measuring
-the bandwidth, which is currently not good enough yet. This needs to
-use a similar method as in "window" algorithm, that is, start with
-equal round-robin and then perform actively a measurement and update
-the cost of sending by assigning so much of a share of the signal
-bitrte as it is represented by the share of the link in the sum of
-all maximum bandwidth values from every link.
+There are possible also other methods and algorithms in the future.
```

Comment:
```suggestion
The addition of other methods and algorithms is permitted.
```

Reply:
I'd still keep "possible in future". What I meant here is that future changes in the SRT source code may be done to add new algorithms.

Reply:
Adding temporal qualifiers like "currently" or "in the latest version" or "in the future" is usually not good practice. As time goes by, the context is lost. I don't think it's necessary to make promises here.